integrations/extensions/starter-kits/zendesk-guide/README.md

```diff
@@ -0,0 +1,133 @@
+# Zendesk Guide article search
+
+## Background
+
+This is a starter kit for accessing the Zendesk "Search Articles" API, which is part of the Zendesk Guide feature.  It is one of two starter kits we have for working with Zendesk.  The other is the [Zendesk Support](https://github.com/watson-developer-cloud/assistant-toolkit/tree/master/integrations/extensions/starter-kits/zendesk-support) starter kit, which enables submitting and querying support tickets for the Zendesk Support feature.
+
+The OpenAPI spec in this starter kit includes the following endpoint:
+
+- `GET /api/v2/help_center/articles/search`: Search for articles relevant to a given query.
+
+That endpoint is described in detail at https://developer.zendesk.com/api-reference/help_center/help-center-api/search/#search-articles.
+
+## Basic vs Advanced
+The `basic` starter kit implements a single yet useful API call for retrieving articles. If you are new to custom extensions, we recommend you start there to get an idea on how to use custom extensions with skills. The `advanced` folder contains a kit with the more advanced functionality of API parameters for filtering your search results, which you can use as a creative springboard for more complex use cases.
+
+## Pre-Requisite Steps
+
+Follow the steps listed in [Pre-Req: Getting Auth Keys and Configuring Your Server](https://github.com/watson-developer-cloud/assistant-toolkit/tree/master/integrations/extensions/starter-kits/zendesk-support#pre-req-getting-auth-keys-and-configuring-your-server) to configure Zendesk to let you access its APIs.
```

Comment:
Maybe need to update the info there which specifies where to put the auth in the UI. (if internal account vs external account)

Reply:
Where to put the auth in the UI is documented here: [https://github.com/watson-developer-cloud/assistant-toolkit/tree/master/integrations/extensions/starter-kits/zendesk-support#pre-req-ge[…]our-server](https://github.com/watson-developer-cloud/assistant-toolkit/tree/master/integrations/extensions/starter-kits/zendesk-support#pre-req-getting-auth-keys-and-configuring-your-server)

Reply:
I could just see a gif for creating a token and the next one is for adding a user to the zenhub account. I did not see a mention of adding username- password here: 

<img width="687" alt="Screen Shot 2022-09-07 at 11 39 43 AM" src="https://user-images.githubusercontent.com/23107360/188953917-78e8f7ed-cb31-4ec0-9266-10e7dd208558.png">

Reply:
more details added in the Zendesk starterkit readme to explain using the API key as a password